functions/src/index.ts

```diff
@@ -265,20 +265,24 @@ export const getStatistics = functions.https.onCall(async (data, context) => {
 export const getProfilePicture = functions.https.onRequest(async (req, res) => {
   const provider = req.query?.provider as string | null;
   const channelId = req.query?.channelId as string | null;
-  if (!provider || !channelId) {
+  const login = req.query?.login as string | null;
```

Comment:
The host schema now includes channelId in the data, can you use that field instead?

Reply:
Isn't that the host channel info? The image is for the user that hosted

Reply:
No channelId should be the hoster's channel. At least if it's not then that's a bug 😓

Reply:
thats confusing [considering](https://github.com/twitch-js/twitch-js/blob/476ff0ab0e5841836b49abb76bd27f90e3cb556a/src/Chat/__tests__/__snapshots__/chat.test.ts.snap#L1298)

Reply:
Oh wait no you're right, the channelId is the hosted channel id. I think we should update the code though to include the hoster's channel id though: https://github.com/muxable/rtchat/blob/main/agent/src/agents/twitch.ts#L92

This would make the event consistent with the other ones

Reply:
So, call `https://api.twitch.tv/helix/users` when the agent receives the event to get the user id?

Reply:
Yes